notebooks/demo_001_latticemaze.ipynb

```diff
@@ -0,0 +1,362 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.imshow(maze.as_img(), cmap=\"gray\", vmin=-1, vmax=1)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It may not be immediately obvious how this image represents a 4x4 maze. Isn't it a 9x9 grid?\n",
+    "\n",
+    "Remember maze is actually repressented as a lattice of nodes, where each pair of nodes may be connected by an edge:\n",
+    "\n",
+    "```\n",
+    "N   N - N - N\n",
+    "|           |\n",
+    "N - N - N - N\n",
+    "            |\n",
+    "N - N   N - N\n",
+    "|       | \n",
+    "N - N - N - N\n",
+    "```\n",
+    "\n",
+    "Each square in the visualisation, may represent a border, a node, a connection, or a wall. In each direction, there are always \n",
```

Comment:
I find it a bit confusing to look at the image and have connections represented as white squares that look identical to nodes. It seems like it would make more sense to not render connections. I assume the reason we're doing that is because the walls (non-connections) take up a full square. I wonder if an alternate version of `as_img()` with thin walls and no connection squares would be clearer... 🤔

Reply:
I also found this confusing. New issue created: https://github.com/AISC-understanding-search/maze-transformer/issues/84